### ISSUE TITLE
No SMS received for first time register

### ISSUE THREAD

**Issue by leongjs98** [forkgram/TelegramAndroid#197]
I am a new Telegram user so I had to register with my phone number. Telegram was supposed to send me an SMS to for signing up, but SMS only works with official Telegram client on google

> Check your Telegram messages
> We've sent the code to the Telegram app for PHONE_NUMBER on your other device.

My suggestion is to just add another sentence.
> If you are registering for the first time, please use the official Telegram app to receive the SMS. This is due to API changes by the Telegram organization

This issue is also discussed in [Telegram-FOSS](https://github.com/Telegram-FOSS-Team/Telegram-FOSS/)

**Comment by paulfertser** [comment]
Good idea, Forkgram should indeed just say it clearly there that neither registration nor changing of the phone number is possible with any non-proprietary client.
@23rd , please consider just adding your own screen instead to all the affected actions. I also ask you to mention that even the proprietary client isn't going to work when SafetyNet isn't working :E

**Comment by ArenaL5** [comment]
Duplicate of #150.
